usr/src/uts/i86pc/os/intr.c

```diff
@@ -471,6 +472,7 @@
 #include <sys/hypervisor.h>
 #endif
 
+boolean_t kernel_preempt_enable = B_FALSE;
```

Comment:
It's no burden for me to wait on his input, and I'll have to wait for somebody's input regardless 😛 , so there's no need for him to interrupt his vacation (unless he wants to check it out).